pkg/verify/verify.go

```diff
@@ -29,6 +30,8 @@ type Interface interface {
 	// Verify should return nil if the provided release digest has suffient signatures to be considered
 	// valid. It should return an error in all other cases.
 	Verify(ctx context.Context, releaseDigest string) error
+
+	Signatures() map[string][][]byte
```

Comment:
Does this need to be part of the verification interface?  Can't `oc` use our `ReleaseVerifier` type directly instead of going through the opaque `verify.Interface`?

Reply:
Good catch. Misstep when I wrapped it in a method in oc. Fixed in [f16cb2d](https://github.com/openshift/cluster-version-operator/pull/413/commits/f16cb2d3f4f0f68ba83532d9090ce64f51dd5b94).